src/compiler/compile/render_dom/wrappers/Element/Attribute.ts

```diff
@@ -204,7 +204,17 @@ export default class AttributeWrapper extends BaseAttributeWrapper {
 		if (this.is_input_value) {
 			const type = element.node.get_static_attribute_value('type');
 
-			if (type === null || type === '' || type === 'text' || type === 'email' || type === 'password') {
+			if (
```

Comment:
I'm using [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#input_types) as a reference for input text types. 
I've ignored DateTime as it's deprecated. Should we add it to support older browsers?